composer.json

```diff
@@ -20,19 +20,19 @@
         }
     ],
     "require": {
-        "php": "^7.1|^8.0",
-        "psr/log": "^1.0|^2.0|^3.0",
-        "firebase/php-jwt": "^3.0|^4.0|^5.0",
+        "php": "^7.4|^8.0",
+        "psr/log": "^1.1|^2.0|^3.0",
+        "firebase/php-jwt": "^5.0",
         "psr/http-message": "^1.0",
-        "tuupola/http-factory": "^0.4.0|^1.0.2",
-        "tuupola/callable-handler": "^0.3.0|^0.4.0|^1.0",
+        "tuupola/http-factory": "^1.4",
+        "tuupola/callable-handler": "^1.1",
```

Comment:
Is there a reason to bump the minimum version of dependencies such as this one? Going for latest is ok for dev dependencies but tends to lead to dependency hell in normal dependencies.

Reply:
@tuupola my main reason behind this was the newer version supports PHP 8, so it would be less of a hassle to upgrade in the future.